### DISCUSSION TITLE
Tariff comparison - not working properly for me (I think !)

### DISCUSSION THREAD

**Discussion by trevorwarwick** [BottlecapDave/HomeAssistant-OctopusEnergy#987] [Q&A]
I'm on the standard Variable tariff, and wanted to compare whether I'd be better or worse off on the Go tariff.  

So I've set up a Tariff Comparison called "go" specifying my electricity meter, and  product code  GO-VAR-22-10-14 and Tariff code E-1R-GO-VAR-22-10-14-H .  

This has successfully created a couple of entities, the interesting one is "go Previous Accumulative Cost Override Electricity (**Meter/Details removed**)". This entity has a value £5.93 that is very similar to the cost for my actual tariff on the day I set it up, but it hasn't changed at all since then, it's just a flat line. I'd expect it to change daily, shadowing the actual cost.

I had disabled a few entities in the main integration since they're not relevant for my normal use, so I wondered if that might be having any effect ?  Any other suggestions ?

**Comment by BottlecapDave** [comment]
Hello and sorry you're having issues. The cost override sensor should update every time the previous consumption data updates. Does your main previous cost sensor still update? Do you have any related errors in your logs? What does the data last retrieved attribute state on the sensor? Have you tried a different tariff to see if there's something wrong with that tariff?

**Comment by trevorwarwick** [comment]
Thanks for the quick reply.

The main integration is working fine, I can see hourly usage.

The Last Data Retrieved date is the date/time that I set it up, which seems consistent with it showing a constant value. 

Nothing obvious in the logs.

**Comment by trevorwarwick** [comment]
This issue persisted until I installed the latest HA update (2024.9.1) a couple of days ago. Since then, it seems to be working as expected, and the Go tracker tariff is showing a value that's closely related to my current one.
